snuba/admin/clickhouse/common.py

```diff
@@ -25,7 +25,11 @@ class InvalidStorageError(SerializableException):
 
 
 def is_valid_node(host: str, port: int, cluster: ClickhouseCluster) -> bool:
-    nodes = [*cluster.get_local_nodes(), cluster.get_query_node()]
+    nodes = [
+        *cluster.get_local_nodes(),
+        cluster.get_query_node(),
+        *cluster.get_distributed_nodes(),
```

Comment:
if there is no `__distributed_cluster_name` is this going to error for some clusters?

Reply:
Good point. It supposed to be that if its None then single_node must be True. But I think they are not doing this in Ops repo. Maybe we should change this assert to warn instead and return an empty list.

Reply:
>  Maybe we should change this assert to warn instead and return an empty list.

yeah I think this makes sense to me

Reply:
Ok. changed it log and return an empty list. Also I set the log level to error rather than warn to bother people to fix their configuration.